projects/js-packages/components/components/record-meter-donut/index.tsx

```diff
@@ -40,21 +54,26 @@ const DonutMeter: React.FC< DonutMeterProps > = ( {
 	backgroundColor = '#00BA37', // jetpack green fallback
 	thickness = '3.5',
 	donutWidth = '64px',
+	title = '',
```

Comment:
What do you think if we do not default title and description? because empty string doesn't make much sense if devs choose to ignore the parameters.

Reply:
Yeah, I was wondering about this. Maybe if the values are not set, we could remove the a11y markup? I imagine if we use this as part of a composite component, the a11y fields might be redundant.

I'll have another look at this.